### ISSUE TITLE
School Ratings: Create usability session test data

### ISSUE THREAD

**Issue by dneel-bah** [department-of-veterans-affairs/va.gov-team#13858]
As a development team member, I need to mock up school ratings data so that the newly implemented functionality can be tested without waiting on live ratings data to come available.

## Assumptions:
1. The data file format matches that outlined in #12962.

## Acceptance Criteria
1. A "school_ratings.csv" populated with test data is created and attached to this story.

### Notes:
- Include institutions with caution flags
- Include institutions without caution flags
- Include institutions without ratings
- Include institutions with less than the minimum number of ratings (5)
- Include institutions with the exact number of minimum ratings (5)
- Include institutions with more than the minimum number of ratings (5)
- Include ratings where not all categories were rated

## Supporting Artifacts:
1. N/A

**Comment by dmccurdybah** [comment]
Some questions:

- How many institutions do we need test data for?
- Are there any specifics we need to know about this test data or is it a random number of ratings with a random spread of rating values?

**Comment by dneel-bah** [comment]
UX team has created test data for usability testing; this story is no longer needed.
